core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -463,39 +463,47 @@ trait EntityComponent {
     def cloneAllEntities(sourceWorkspaceContext: SlickWorkspaceContext, destWorkspaceContext: SlickWorkspaceContext): ReadWriteAction[Int] = {
       val allEntitiesAction = listActiveEntitiesAllTypes(sourceWorkspaceContext)
 
-      allEntitiesAction.flatMap(cloneEntities(destWorkspaceContext, _))
+      allEntitiesAction.flatMap(cloneEntities(destWorkspaceContext, _, Seq.empty))
     }
 
     def batchInsertEntities(workspaceContext: SlickWorkspaceContext, entities: Seq[EntityRecord]): ReadWriteAction[Seq[EntityRecord]] = {
       if(!entities.isEmpty) {
         workspaceQuery.updateLastModified(workspaceContext.workspaceId) andThen
-          insertInBatches(entityQuery, entities) andThen selectEntityIds(workspaceContext, entities)
+          insertInBatches(entityQuery, entities) andThen selectEntityIds(workspaceContext, entities.map(e => (e.entityType, e.name)))
       }
       else {
         DBIO.successful(Seq.empty[EntityRecord])
       }
     }
 
-    def selectEntityIds(workspaceContext: SlickWorkspaceContext, entities: Seq[EntityRecord]): ReadAction[Seq[EntityRecord]] = {
+    def selectEntityIds(workspaceContext: SlickWorkspaceContext, entities: Seq[(String, String)]): ReadAction[Seq[EntityRecord]] = {
       val entitiesGrouped = entities.grouped(batchSize).toSeq
 
       DBIO.sequence(entitiesGrouped map { batch =>
-        EntityRecordRawSqlQuery.action(workspaceContext.workspaceId, batch.map(_.toReference))
+        EntityRecordRawSqlQuery.action(workspaceContext.workspaceId, batch.map{ case (entityType, name) => AttributeEntityReference(entityType, name) })
```

Comment:
I think this will just be `batch` if you change the `entities` param to `Seq[AttributeEntityReference]`